app/models/crime_application.rb

```diff
@@ -135,6 +135,10 @@ def outgoings_details
     @outgoings_details ||= OutgoingsDetailsPresenter.present(self[:means_details].outgoings_details)
   end
 
+  def properties
+    @properties ||= PropertiesPresenter.present(self[:means_details].capital_details&.properties || [])
```

Comment:
I think it's an empty array by default so you don't need the `|| []`
If a user selects no to say their client doesn't have any assets, what is meant to be displayed in review? I don't think there is a design in the review section

Reply:
Actually i found this example for national savings certificate: https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=11311-44907&mode=design&t=gFQeEeNApBZHvUQ7-0

<img width="747" alt="Screenshot 2024-03-21 at 13 38 01" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/51047911/512635e4-e6c5-447c-8f1b-add272ae3101">

Reply:
empty assets need to confirm with designers